Mathlib/CategoryTheory/Limits/Preserves/Finite.lean

```diff
@@ -106,6 +107,13 @@ instance compPreservesFiniteProducts (F : C ⥤ D) (G : D ⥤ E)
 noncomputable instance (F : C ⥤ D) [PreservesFiniteLimits F] : PreservesFiniteProducts F where
   preserves _ _ := inferInstance
 
+noncomputable instance (priority := 100) (J : Type u) [Fintype J] (F : C ⥤ D)
```

Comment:
Can it be `Finite J` instead?

Reply:
Yes! In fact we already had this for coproducts. I've changed it so products/coproducts are in sync.